dockerize.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+IMAGE_NAME="dashboard"

```

Comment:
image names get messy without prefixes. `boresha/dashboard` would  be better.

Also, it's more normal to have a new line after the shebang at the top.

Reply:
I agree that having a prefix would help, but my concern was with having the company name in the code just in case the name changes (ie. like how we switched from lishabora to boresha). That way we won't have to refactor our code in multiple repositories when there's an external change in business.  What are your thoughts on this? Maybe there's another prefix that we could use?

Reply:
We talked about this and decided we can include the company name to the docker image name